alacritty_terminal/src/tty/windows/mod.rs

```diff
@@ -38,20 +38,18 @@ pub fn is_conpty() -> bool {
     IS_CONPTY.load(Ordering::Relaxed)
 }
 
-#[derive(Clone)]
 pub enum PtyHandle {
-    Winpty(winpty::WinptyHandle),
-    Conpty(conpty::ConptyHandle),
+    Winpty(winpty::WinptyAgent),
+    Conpty(conpty::Conpty),
+}
+
+enum PtyResizeHandle {
+    Winpty(winpty::WinptyResizeHandle),
+    Conpty(conpty::ConptyResizeHandle),
 }
 
 pub struct Pty {
     handle: PtyHandle,
-    // TODO: It's on the roadmap for the Conpty API to support Overlapped I/O.
-    // See https://github.com/Microsoft/console/issues/262
-    // When support for that lands then it should be possible to use
-    // NamedPipe for the conout and conin handles
```

Comment:
Where did this comment go? It seems like that would improve the ConPTY API so tracking it makes sense.

Unless we have an issue on GitHub at least, I'm all for opening an issue instead of tracking it in code.

Reply:
There's two nearly identical copies of this comment; I removed this and left the other one: https://github.com/jwilm/alacritty/blob/5f4f253cacbe29c1b473004cfeeb1b2caf25f913/alacritty_terminal/src/tty/windows/mod.rs#L121

We also have an issue: #1905